syntax = "proto3";
package common;

enum ResultType {
    SUCCESS = 0;
    FAIL = 1;
}

message ValueFunction {
    string data = 1;
}

message ValueFuncUpdate {
    string data = 1;
}

message Policy {
    string data = 1;
}

message PolicyUpdate {
    string data = 1;
}

message Observation {
    string data = 1;
}

message Action {
    string data = 1;
}